src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const BASE_URL = 'https://mate.academy/students-api';
+
+// returns a promise resolved after a given delay
```

Comment:
Add a space after the double slashes to follow the code commenting guidelines: '// returns a promise resolved after a given delay'